llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -27569,6 +27576,12 @@ void AArch64TargetLowering::ReplaceNodeResults(
     if (SDValue Res = LowerVECTOR_COMPRESS(SDValue(N, 0), DAG))
       Results.push_back(Res);
     return;
+  case ISD::PARTIAL_REDUCE_UMLA:
+  case ISD::PARTIAL_REDUCE_SMLA: {
+    if (SDValue Res = LowerPARTIAL_REDUCE_MLA(SDValue(N, 0), DAG))
+      Results.push_back(Res);
+    return;
+  }
```

Comment:
Sorry just noticed this, but Is this code actually used? (normally this is only needed when the result type is not legal, but the input is).